data/abilities/credential-access/1b4fb81c-8090-426c-93ab-0a633e7a16a7.yml

```diff
@@ -21,3 +21,15 @@
           Remove-NetEventSession -Name "Capture"
           Remove-Item $ENV:UserProfile\Desktop\pcap.etl
           Remove-Item $ENV:UserProfile\Desktop\pcap.cab
+    darwin:
+      sh:
+        command: |
+          tcpdump -i en0 & sleep 5; kill $!
+        visibility:
+          score: 55
```

Comment:
how are you measuring visibility?

Reply:
0 to 100 scale. all abilities get 50 by default because it's "50/50" whether they are visible. this block shows a first example of how they can be modified. I'm unsure whether I like these here or separate from the YML files themselves (since the adjustments are subjective)

Reply:
sweet thanks